src/components/App.tsx

```diff
@@ -124,6 +126,33 @@ class App extends React.Component<Props, State> {
                 />
               )}
             />
+            <Route
+              path="/sivut/:page/:page/:page"
+              render={ (props) => (
+                <PostPage
+                  lang={ language }
+                  slug={ this.pathToSlug(props.location.pathname) }
+                  mainPageSlug={ this.pathToTitle(props.location.pathname) }
+                  locationPath={ props.location.pathname }
+                />
+              )}
+            />
+            <Route
+              path="/:page"
+              render={ (props) => {
+                // Check for special pages
+                if (/^\/event\/|\/announcements\/$|^\/news\/$|^\/jobs\/$|^\/sivut\//.test(props.location.pathname)) {
```

Comment:
Should there be dollar sign in` /^\/event\/$` ?

Reply:
Dollar sign in regex means the end of the string. Event path includes event page name after "/event/" for example "/event/eventName" thats why there is no dollar sign. Same logic applies to "/sivut/" which can contain multiple page names after it.